tools/topology/sof-icl-rt711-rt1308-rt715-hdmi.m4

```diff
@@ -38,39 +38,39 @@ dnl     dai type, dai_index, dai format,
 dnl     dai periods, pcm_min_rate, pcm_max_rate,
 dnl     pipeline_rate, time_domain)
 
-# Low Latency playback pipeline 1 on PCM 0 using max 2 channels of s32le.
+# Low Latency playback pipeline 1 on PCM 0 using max 2 channels of s24le.
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
 PIPELINE_PCM_DAI_ADD(sof/pipe-volume-playback.m4,
-	1, 0, 2, s32le,
-	1000, 0, 0, ALH, 2, s32le, 3,
+	1, 0, 2, s24le,
```

Comment:
Maybe I'm wrong here, but I thought s24le is transferred in 32 bit container anyway.
And what does it exactly mean, that bandwidth is not enough? It sounds kind of weird, that we don't have enough resources for two streams.

Reply:
It's a result of the 6MHz clock and 2 columns/125 rows. Since the allocation only uses the second column, we have 125 bits per frame to store 128 audio bits. That doesn't work too well.
And yes at the ALH level, all DMA transfers are left aligned 32-bits, but here we ignore the 8 LSBs during the transmission.